fixtures/fixture_generator/Sources/FixtureGenerator/Templates/ManifestTemplate.swift

```diff
@@ -0,0 +1,70 @@
+
+import Foundation
+
+class ManifestTemplate {
```

Comment:
we can also leverage [GYB](https://nshipster.com/swift-gyb/) to generate boilerplate code, apple swift source code uses it in many places.